api/src/Common/Models/Race.cs

```diff
@@ -21,6 +21,10 @@ public class Race : IModel
         [Required]
         public string Distance { get; set; }
 
+        public bool Public { get; set; }
```

Comment:
Do you think this change goes tied with adding the org info for a race? It feels like races are public in the context of an org.

Reply:
no, this is intentional. This indicates if a race is currently shown on the submisisons form

Reply:
yes you are correct that an org is supposed to decide if the race is publicly visible. We need to make races specific to an org later. But we need this field now for the feature of managing which races are shown.